camelot/view/controls/editors/floateditor.py

```diff
@@ -207,6 +204,9 @@ def set_field_attributes(self, **kwargs):
         precision = kwargs.get('precision', 2)
         if spinBox.decimals() != precision:
             spinBox.setDecimals( precision )
+        minimum = kwargs.get('minimum') if kwargs.get('minimum') is not None else constants.camelot_minfloat-1
```

Comment:
je kan dit best afhandelen in de FloatDelegate : 

- dan moet die logica niet in de editor lopen (gui thread)
- is die logica ook bruikbaar voor andere editors dan deze (bvb qml editor in polapp)